vtm/src/org/oscim/tiling/TileSource.java

```diff
@@ -81,7 +81,6 @@ public String getName() {
         }
     }
 
-    // FIXME Same as BuildingLayer.MAX_ZOOM
     public static final int MAX_ZOOM = 17;
```

Comment:
Here can use `BuildingLayer.MAX_ZOOM`.
And have the FIXME declaring that after buildings overzoom implementation, eventually the `Viewport.MAX_ZOOM_LEVEL` should be default.